### ISSUE TITLE
Debug information

### ISSUE THREAD

**Issue by ci70** [MahdiSafsafi/UnivDisasm#5]
Hi!

Can you make a demo something like this for both 32&64?

http://i.imgur.com/gNhiwAo.png

**Comment by MahdiSafsafi** [comment]
Hi joeparkerz,

Are you talking about UnivDisasm GUI ?

Regards,
Mahdi.

**Comment by ci70** [comment]
Yes and generally. Hopefully it is possible to parse so I can analylze it. For Delphi EXE that is.

**Comment by MahdiSafsafi** [comment]
Sorry,It's mostly impossible to integrate such feature.The GUI is just a demo to show UnivDisasm capability.

Mahdi.

**Comment by ci70** [comment]
Oh then it doesn't help much for Delphi projects. Too bad! :)
